charts/renga/charts/gitlab/templates/deployment.yaml

```diff
@@ -59,7 +59,7 @@ spec:
           containerPort: 8105
         livenessProbe:
           httpGet:
-            path: /gitlab/help
```

Comment:
could we template this? Or use an environment variable for the prefix? That would allow someone to run it on a path instead of subdomain if they wanted to.